.github/workflows/functional-test-cloud.yaml

```diff
@@ -431,9 +437,9 @@ jobs:
       - name: Login to Azure
         uses: azure/login@v2
         with:
-          client-id: ${{ secrets.AZURE_SP_TESTS_APPID }}
-          tenant-id: ${{ secrets.AZURE_SP_TESTS_TENANTID }}
-          subscription-id: ${{ secrets.AZURE_SUBSCRIPTIONID_TESTS }}
+          client-id: ${{ secrets.AZURE_TEST_APPID }}
+          tenant-id: ${{ secrets.AZURE_TEST_TENANTID }}
+          subscription-id: ${{ secrets.AZURE_TEST_SUBSCRIPTIONID }}
```

Comment:
I added these in the repo secrets, we should move them to org secrets

Reply:
before we merge, I will figure this out with management. safe to assume that these will be populated with the correct values.

Reply:
Why do we need to use new clientID, tenantID, and subscriptionID?

Reply:
we don't - I added them for testing. we can use the same variables but I would need to edit the existing app we use. I'll talk to mgmt